homeworks/serhii.noskov_SerhiiNSN/GamesProject/login.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link
+            href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic"
+            rel="stylesheet"
+        >
+        <link rel="stylesheet" href="css/style.css">
+        <title>GamesProject</title>
+    </head>
+
+    <body>
+        <header class="header">
+            <div class="header__container">
+                <a class="header__logo" href="#">
+                    <img src="img/logo.svg" alt="logo">
+                </a>
+                <nav class="nav">
+                    <ul class="header__menu">
+                        <li class="menu__list">
+                            <a href="games.html" class="menu__link">Games</a>
+                        </li>
+                        <li class="menu__list">
+                            <a href="index.html" class="menu__link">About</a>
+                        </li>
+                        <li class="menu__list">
+                            <a href="login.html" class="menu__link">Login</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <main class="main">
+            <section class="login">
+                <div class="login__container">
+                    <div class="form__inner">
+                        <div class="login__tittle">Login page</div>
+                        <form action="success-login.html" class="email-form">
+                            <div class="form__area">
+                                <label for="email__input" class="email__text"
+                                >Email</label
+                                >
+                                <input
+                                    type="email"
+                                    required
+                                    placeholder="Enter your email address"
+                                    class="email__input"
+                                >
+                                <label
+                                    for="password__input"
+                                    class="email__password"
+                                >Password</label
+                                >
+                                <input
+                                    type="password"
+                                    required
+                                    placeholder="Enter your password"
+                                    class="password__input"
+                                >
+                            </div>
+                            <div class="verefication">
+                                <input
```

Comment:
You've lost the checkbox here, and the form can be submitted despite this checkbox 
<img width="801" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/eacf735e-d475-4687-8561-d1d5f3f94f52">